Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -12,9 +12,12 @@ import SwiftUI
 
 /// Displays the content of a `Chat` message in a message bubble
 public struct MessageView: View {
-    let chat: Chat
-    let hideSystemMessages: Bool
+    public enum Defaults {
+        public static let hideMessagesWithRoles: Set<Chat.Role> = [.system, .function]
+    }
     
+    let chat: Chat
```

Comment:
You can make this property private:
```suggestion
    private let chat: Chat
```